src/themes/teams/components/Icon/svg/icons/index.ts

```diff
@@ -34,6 +41,9 @@ export default {
   format,
   giphy,
   highlight,
+  bullets,
```

Comment:
it is better to sort these keys alphabetically, so it will be easier to search icon with specific name

Reply:
Done